src/components/Guilds/ProposalTypes/index.tsx

```diff
@@ -0,0 +1,224 @@
+const CloseIcon = styled(FiX)`
+  color: ${({ theme }) => theme.colors.text};
+  height: 1.5rem;
+  width: 1.5rem;
+
+  &:hover {
+    cursor: pointer;
+    opacity: 0.6;
+  }
+`;
+
+const Wrapper = styled(Flex)`
+  max-width: 386px;
+  @media only screen and (max-width: 768px) {
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    z-index: 1000;
+  }
+`;
+const Container = styled(Flex)`
+  border: 1px solid ${({ theme }) => theme.colors.primary};
+  margin: 10px 0;
+  width: 100%;
+
+  @media only screen and (max-width: 768px) {
+    width: 90%;
+  }
+`;
+
+const PaddingWrapper = styled(Flex)`
+  margin: 24px;
+  width: 100%;
+`;
+
+const ContentWrapper = styled(Flex)`
+  height: 100%;
+  justify-content: flex-start;
+`;
+
+const ProposalTypeButton = styled(Button)`
+  width: 90%;
+  height: 40px;
+  margin: 6px 0;
+  flex-direction: row;
+  justify-content: left;
+`;
+
+const ContainerHeader = styled(Flex)`
+  width: 90%;
+  justify-content: initial;
+  flex-direction: row;
+  margin: 6px 0;
+`;
+
+const Footer = styled(Flex)`
+  flex-direction: row;
+  justify-content: right;
+  width: 100%;
+
+  @media only screen and (max-width: 768px) {
+    border-top: 1px solid black;
+    justify-content: center;
+    align-items: center;
+    border-radius: 0;
+    padding: 20px;
+  }
+`;
+
+const ButtonFooter = styled(Button)`
+  @media only screen and (max-width: 768px) {
+    width: 90%;
+  }
+`;
+
+const Header = styled(Flex)`
+  align-items: flex-start;
+  width: 100%;
+
+  @media only screen and (max-width: 768px) {
+    border-radius: 0;
+    border-bottom: 1px solid black;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 20px;
+    width: 90%;
+  }
+`;
+
+const HeaderWrap = styled(Flex)`
+  flex-direction: row;
+`;
+
+const TitleWrapper = styled(Flex)`
+  width: 100%;
+  align-items: flex-start;
+  margin-top: 10px;
+
+  @media only screen and (max-width: 768px) {
+    width: 90%;
+  }
+`;
+
+interface ProposalTypeDescriptionProps {
+  title: string;
+  description: string;
+  icon?: React.FC<React.SVGProps<SVGSVGElement>>;
+  onChainAction: boolean;
+}
+
+const ProposalTypeDescription: React.FC<ProposalTypeDescriptionProps> = ({
+  description,
+  title,
+  onChainAction,
+}) => {
+  return (
+    <Container>
+      <PaddingWrapper>
+        <ContainerText variant="bold">{title}</ContainerText>
+        <ContainerText variant="medium">{description}</ContainerText>
+        <ContainerText variant="medium" color="grey">
+          {onChainAction ? 'On-chain Action' : 'No on-chain action'}
+        </ContainerText>
+      </PaddingWrapper>
+    </Container>
+  );
+};
+
+interface ProposalTypesProps {
+  data: ProposalTypeDescriptionProps[];
+}
+const ProposalTypes: React.FC<ProposalTypesProps> = ({ data }) => {
+  const history = useHistory();
+  const location = useLocation();
+
+  const continueUrl = location.pathname.replace(
+    '/ProposalType',
+    '/create/proposal_type'

```

Comment:
Could we make this add the actual proposal type to the URL?

Reply:
I was not sure whether or not each proposal type would have a different URL and template? or if it would all point to the same URL with predefined components?.

I'd rather leave this change outside of this scope.